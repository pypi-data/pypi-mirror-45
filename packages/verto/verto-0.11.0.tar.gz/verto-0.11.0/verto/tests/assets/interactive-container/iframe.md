{interactive slug="binary-cards" type="iframe" parameters="digits=5&start=BBBBB"}
