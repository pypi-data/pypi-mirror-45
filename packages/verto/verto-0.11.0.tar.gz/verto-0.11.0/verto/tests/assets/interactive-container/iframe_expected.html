<p>{interactive slug="binary-cards" type="iframe" parameters="digits=5&amp;start=BBBBB"}</p>
